frontend/src/container/PublicProfile.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { translate } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import {
+  BREADCRUMBS_TYPE,
+  CUSTOM_EVENT,
+  PAGE,
+  serialize,
+  TracimComponent
+} from 'tracim_frontend_lib'
+import {
+  newFlashMessage,
+  setBreadcrumbs
+} from '../action-creator.sync.js'
+import { getUserPublicInformation } from '../action-creator.async'
+import { serializeUserProps } from '../reducer/user.js'
+import ProfileMainBar from '../component/PublicProfile/ProfileMainBar.jsx'
+
+export class PublicProfile extends React.Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      displayedUser: undefined,
+      coverImageUrl: undefined
+    }
+
+    props.registerCustomEventHandlerList([
+      { name: CUSTOM_EVENT.ALL_APP_CHANGE_LANGUAGE, handler: this.handleAllAppChangeLanguage }
+    ])
+  }
+
+  handleAllAppChangeLanguage = data => {
+    console.log('%c<Profile> Custom event', 'color: #28a745', CUSTOM_EVENT.ALL_APP_CHANGE_LANGUAGE, data)
+    this.buildBreadcrumbs()
+  }
+
+  componentDidMount () {
+    this.getUser()
+  }
+
+  buildBreadcrumbs = () => {
+    const { props, state } = this
+
+    props.dispatch(setBreadcrumbs([{
+      link: (
+        <Link to={PAGE.HOME} title={props.t('Home')} className='primaryColorFont primaryColorFontDarkenHover'>
+          <i className='fa fa-home' />{props.t('Home')}
+        </Link>
+      ),
+      type: BREADCRUMBS_TYPE.CORE,
+      label: props.t('Home')
+    }, {
+      link: <Link to={PAGE.SEARCH_RESULT}>{props.t("{{user}}'s profile", { user: state.displayedUser.publicName })}</Link>,
+      type: BREADCRUMBS_TYPE.CORE,
+      label: props.t("{{user}}'s profile", { user: state.displayedUser.publicName })
+    }]))
+  }
+
+  getUser = async () => {
+    const { props } = this
+    const userId = props.match.params.userid
+
+    const fetchGetUser = await props.dispatch(getUserPublicInformation(userId))
+
+    switch (fetchGetUser.status) {
+      case 200:
+        this.setState({
+          displayedUser: serialize(fetchGetUser.json, serializeUserProps),
+          coverImageUrl: 'default'
+        })
+        this.buildBreadcrumbs()
+        break
+      case 400:
+        switch (fetchGetUser.json.code) {
+          case 1001:
+            props.dispatch(newFlashMessage(props.t('Unkown user')))
+            props.history.push(PAGE.HOME)
+            break
+          default:
+            props.dispatch(newFlashMessage(props.t('Error while loading user')))
+            props.history.push(PAGE.HOME)
+        }
+        break
+      default:
+        props.dispatch(newFlashMessage(props.t('Error while loading user')))
+        props.history.push(PAGE.HOME)
+    }
+  }
+
+  render () {
+    const { props, state } = this
+
+    return (
+      <div className='tracim__content fullWidthFullHeight'>
+        <div className='tracim__content-scrollview'>
+          <div className='profile__cover'>
```

Comment:
is there a reason why you are not using PageWrapper ?

Reply:
yes, PageWrapper has a margin (or a padding) and in this case the cover image is "together" with the rest of the page. Maybe I can make a new component for the content part and use PageWrapper at this component.

Reply:
Actually, I don't think it will work, if I remember correctly it is required a PageTitle on a PageWrapper

Reply:
indeed the layout is different for PublicProfile